packages/documentation-site/patternfly-docs/content/developer-resources/patternfly-accessibility.md

```diff
@@ -3,34 +3,35 @@ id: PatternFly's accessibility
 section: accessibility
 ---
 
-As a design system, PatternFly provides accessible building blocks to work with. We do our best to cover all areas that are within our control and take at least some of the accessibility work off your hands.
+# PatternFly components accessibility 
```

Comment:
This seems like it'd be better as a level 2 heading. Also, could we omit the "PatternFly" from these two headings? Since we already have the "PatternFly's accessibility" heading, shortening these to "Component accessibility" and "Accessibility standards"